measureme/src/stringtable.rs

```diff
@@ -187,11 +180,10 @@ impl<'s> StringComponent<'s> {
                 &mut bytes[s.len()..]
             }
             StringComponent::Ref(string_id) => {
-                assert!(string_id.0 == string_id.0 & STRING_ID_MASK);
-                let tagged = string_id.0 | (1u32 << 31);
-
-                &mut bytes[0..4].copy_from_slice(&tagged.to_be_bytes());
-                &mut bytes[4..]
+                assert!(STRING_REF_ENCODED_SIZE == 5);
+                bytes[0] = STRING_REF_TAG;
+                &mut bytes[1..5].copy_from_slice(&string_id.0.to_le_bytes());
+                &mut bytes[5..]
```

Comment:
Oh, I see it's for this. How do you feel about doing this instead? 

```suggestion
                &mut bytes[1..STRING_REF_ENCODED_SIZE].copy_from_slice(&string_id.0.to_le_bytes());
                &mut bytes[STRING_REF_ENCODED_SIZE..]
```